backend/http_settings_test.go

```diff
@@ -149,3 +149,18 @@ func TestParseHTTPSettings(t *testing.T) {
 		})
 	})
 }
+func TestParseHTTTPSettingsWithInvalidTLSOptions(t *testing.T) {
+	t.Run("ignore TSL options if they are not boolean", func(t *testing.T) {
+		jsonData := `{
+							"tlsAuth": "true",
+							"tlsAuthWithCACert": "true",
+							"tlsSkipVerify": "true"
+							}`
+		secureData := map[string]string{}
+		res, err := parseHTTPSettings([]byte(jsonData), secureData)
+		require.NoError(t, err)
+		require.False(t, res.TLSClientAuth)
+		require.False(t, res.TLSAuthWithCACert)
+		require.False(t, res.TLSSkipVerify)
+	})
+}
```

Comment:
nit: is there a positive test also? I mean that tests that boolean work as expected?

Reply:
Yes, in `TestParseHTTPSettings` there are already positive tests for all of these fields